CMakeLists.txt

```diff
@@ -133,6 +133,9 @@ if (FLANN_FOUND)
     include_directories(SYSTEM "${FLANN_INCLUDE_DIRS}")
 endif()
 
+find_package( Eigen REQUIRED )
+include_directories(SYSTEM "${Eigen_INCLUDE_DIRS}")
+
```

Comment:
I think a similar line needs to be added to the OMPL.app project, but I don't have a fork of that.